### PR TITLE
 docs(contributing): add contributing.md to help contributers

### LINKED ISSUE #86: Create a contribution documentation
As a developer I want to have a guideline to create a contribution ;)

Seriously would be nice to have such a thing to make it easier for everybody to contribute to this nice project.

### PR DESCRIPTION
**Pull request by kotzendekrabbe**
Please feel free to give as much feedback as possible + look at gramma etc. thx :)

Fix #86

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- *type: design* - issue for a design todo
+- *type: documentation* - issue for needed, missing or unclear documentation
+- *status: in progress* - issue is currently in progress
+- *status: has PR* - issue has already an open pull request
+- *priority: low* - issue which is a nice to have, like a backlog issue
+- *priority: mid* - issue needed in the near future, but not super urgent
+- *priority: high* - issue must be resolved as soon as possible
+
+### Working on issues
+Issues should be provided as branches starting with "feat/", as pull requests.
```
Comment (LKuechler) [inline CONTRIBUTING.md]: Would it make more sense to write something like this:
"Pull requests should be provided as branches starting with 'feat/'"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   * backlog board: https://github.com/meetalva/alva/projects/3
+
+
+## Setup for contributers
+To add a contribution to Alva (improve its features, fix issues),
+get the source of both the application itself,
+and the kickstart styleguide (designkit):
+
+* Clone this repository [alva](https://github.com/meetalva/alva): git@github.com:meetalva/alva.git
+* Clone [designkit](https://github.com/meetalva/designkit) repository: git@github.com:meetalva/designkit.git
```
Comment (LKuechler) [inline CONTRIBUTING.md]: I think i should bring more context to this.
The designkit should not really be needed for the development. It is just a example project.

If the contributer already has a lsg that is compatible with alva there is no need to use the designkit.
If you want i can try to rewrite this part

File: CONTRIBUTING.md (hunk 1 above)
Comment (kotzendekrabbe) [reply CONTRIBUTING.md]: as below - only copied this from readme.

Yes, it's more clear.

Also, I would prefer to say - branches should start similar to commit conventions: feat, docs, bug, chore etc.
BUT I don't know what you as project team decided :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+* Run in the designkit repository:
+```shell
+npm i && npm run build
+```
+* Run in the main (Alva) respository:
```
[the remark this answers is not in the document]
Comment (kotzendekrabbe) [reply CONTRIBUTING.md]: this part is only for "contributing" not for modify styling components.
So we only need to show how the contributer setup is , and to be honest - i don't know it.
also a copy & paste text ;)

File: CONTRIBUTING.md (hunk 1 above)
Comment (LKuechler) [reply CONTRIBUTING.md]: I thing its a good idea. But i also have the feeling that the name of a branch is not that important. After the merge no one will look at the name again :)

**Comment by kotzendekrabbe** [comment]
@LKuechler thx for your changes :)

**Pull request merged**